### DISCUSSION TITLE
Algorithmic Fairness Metrics

### DISCUSSION THREAD

**Discussion by patrickm663** [JuliaTrustworthyAI/.github#5] [Ideas]
Hi

Thanks for the work done to bring trustworthy AI to Julia!

I am interested to hear if implementations of algorithmic fairness (equalised odds, demographic parity) are planned or within scope for this project? For instance, porting some functionality from Python's [`fairlearn`](https://fairlearn.org/) library to Julia, while being compatible with Flux.

I am planning to write up a minimal example on implementing fairness metrics in Julia using Flux models so that it may help raise awareness to others of the techniques. I haven't considered a full package, but I am happy to take a stab if it would be useful!

Regards
Patrick

**Comment by pat-alt** [comment]
@patrickm663 I am very sorry I have somehow missed this! That sounds like a great idea, would love to see the write up. I stumbled across this interesting [project](https://github.com/zenna/CounterfactualFairness.jl) by @zenna recently. Perhaps some of that might be useful for what you have in mind @patrickm663?
